src/main/java/animals/controller/Game.java

```diff
@@ -0,0 +1,162 @@
+                    break;
+                case SEARCH_ANIMAL:
+                    System.out.println(getLocalString("enter.animal"));
+                    String userAnimal = SCANNER.nextLine();
+                    kts.getAllFactsAboutAnimal(userAnimal, root);
+                    System.out.println();
+                    break;
+                case STATISTICS:
+                    kts.showStats(fs.load(type, FILE_NAME));
+                    System.out.println();
+                    break;
+                case KNOWLEDGE_TREE:
+                    kts.printTree(fs.load(type, FILE_NAME));
+                    System.out.println();
+                    break;
+                case EXIT:
+                    quit = true;
+                    SCANNER.close();
+                    System.out.println(getLocalString("bye"));
+                    break;
+                case UNDEFINED:
+                    System.out.println(getLocalString("wrong"));
+            }
+        }
+    }
+
+    public void play(Node node) {
+        System.out.println(node.askQ());
+        if (!node.isLeaf()) {
+            if (askYesNo()) {
+                play(node.getYes());
+            } else {
+                play(node.getNo());
+            }
+        } else {
+            if (!askYesNo()) {
+                String firstAnimal = node.getData();
+                System.out.println(getLocalString("give.up"));
+                String userNewAnimal = lr.getAnimal(SCANNER.nextLine().strip().toLowerCase());
+
+                String positive = lr.getStatement(firstAnimal, userNewAnimal);
+                String negative = lr.toNegative(positive);
+                System.out.println(getLocalString("is.correct") + userNewAnimal + "?");
+
+                String learn1 = lr.toAnimalFact(positive, firstAnimal);
+                String learn1Neg = lr.toAnimalFact(negative, firstAnimal);
+                String learn2 = lr.toAnimalFact(positive, userNewAnimal);
+                String learn2Neg = lr.toAnimalFact(negative, userNewAnimal);
+                String question = lr.getQuestion(positive);
+                Node secondNode = new Node(userNewAnimal);
+                if (!askYesNo()) {
+                    node.yes = new Node(node.getData());
+                    node.data = question;
+                    node.no = secondNode;
+                    node.setLeaf(false);
+                    System.out.println(lr.getLearnedPhrase(learn1, learn2Neg, question));
+                } else {
+                    node.no = new Node(node.data);
+                    node.data = question;
+                    node.yes = secondNode;
+                    node.setLeaf(false);
+                    System.out.println(lr.getLearnedPhrase(learn1Neg, learn2, question));
+                }
+            }
+        }
+    }
+
+    public boolean askYesNo() {
+        final var bundle = ResourceBundle.getBundle("application");
```

Comment:
Why `ResourceBundle.getBundle("application")` and not `getLocalString(...)`?
Please check all places where `ResourceBundle.getBundle("application")` is used